DEVX-2225: Convert column concatenation recipe to KT (#660)
confluentinc/kafka-tutorials: _includes/tutorials/ksql-concatenation-columns/ksql/markup/dev/transient-reporting.adoc

> Now we write a query where we select the desired fields but we don't have them listed in a comma separated list.  Instead we use the `+` operator between the fields causting them to get concatenated together.

Reviewer: ```suggestion
Now we write a query to concatenate multiple columns. To achieve this, we will use the `+` operator between the fields in our `SELECT` statement rather than a comma.
```
Reading through this a few times, I thought maybe it was talking about the output of the query being comma separated. Feel free to wordsmith different. I think it would be helpful here to say what you are trying to accomplish (column concat) and how you are doing that (+ instead of ,).

Reviewer 2: ➕ to what @awalther28 wrote.  Explain what you're doing first (which fields, concat, why, etc)...then show.

> Now we write a query where we select the desired fields but we don't have them listed in a comma separated list.  Instead we use the `+` operator between the fields causting them to get concatenated together.

Reviewer: ```suggestion
Now we write a query where we select the desired fields but we don't have them listed in a comma separated list.  Instead we use the `+` operator between the fields causing them to get concatenated together.
```
^ If you don't move forward with my other suggestion, I think you meant 'causing'

[source, sql]
----

SELECT FIRST_NAME + ' ' + LAST_NAME +
       ' purchased ' +
       CAST(NUM_SHARES AS VARCHAR) + <1>
       ' shares of ' +
       SYMBOL AS SUMMARY
FROM ACTIVITY_STREAM
EMIT CHANGES
LIMIT 4;

----

> <1> The NUM_SHARES field is an `INT` so we need to cast it to a `VARCHAR` as `concatenate` only works with `STRING` types

Reviewer: What happens if you don't cast a field to a varchar?

Author: results in an error, I've updated the text to state as much

> NOTE: You can also select fields you don't want to concatenate.  In that case you use a comma to separate the field from those you concatenate together.  For example

Reviewer 2: ```suggestion
NOTE: You can also SELECT fields you don't want to concatenate.  In that case you use a comma to separate the field from those you concatenate.  For example, you can SELECT individual fields `field_1` and `field_2` at the same time that you concatenate `field_3` with `field_4`.
```

> SELECT field_1, field_2, field_3 + field_4

Reviewer 2: ```suggestion
`SELECT field_1, field_2, field_3 + field_4`
```

This query should produce the following output:

+++++
<pre class="snippet"><code class="shell">{% include_raw tutorials/ksql-concatenation-columns/ksql/code/tutorial-steps/dev/expected-transient-reporting.log %}</code></pre>
+++++

> Now that the reporting query works, let's update it to create a continous query for your reporting scenario

Reviewer: ```suggestion
Now that the reporting query works, let's update it to create a continuous query for your reporting scenario
```
